src/routes/docs/products/messaging/send-push-notifications/+page.markdoc

```diff
@@ -397,16 +397,19 @@ Before your app can receive push notifications, you need to request the user for
 Appwrite provides a utility to help request permissions to display notificaitons.
 
 ```swift
-OS.requestPermission(
-    application,
-    OSPermission.notifications,
-    onGranted: {
-        // Handle on permission granted
-    },
-    onDenied: {
-        // Handle on permission denied
+func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
+    ) async -> Bool {
+        // ... other setup logic
+        let center = UNUserNotificationCenter.current()
+
+        do {
+            try await center.requestAuthorization(options: [.alert, .sound, .badge])
+        } catch {
+            // Handle the error here.
+        }
```

Comment:
If we just direct to docs for Android, should we do the same for Apple?

Reply:
Honestly yeah, this bit of code will break 8/10 times because of how their apps are set up.